js/components/utilities/export.js

```diff
@@ -25,15 +25,19 @@ define([
             this.entityId = params.entityId;
             this.isPermittedExport = params.isPermittedExport || (() => false);
             this.exportService = params.exportService;
-
+            this.dirtyFlag = params.dirtyFlag;
+            this.errorMessage = params.errorMessage || 'Export is not permitted';
```

Comment:
I think we'd like to have separate messages in the 2 possible scenarios for this component:

1. You do not have permissions to export 
2. The export is prevented until the calling entity is saved

I'm supportive of passing in an error message as you've done here for item 2 above. In the event that you do not have permissions, the component should explicitly state this.

Reply:
@anthonysena 
So, as far as i understood we can do the following
1) If entity is not saved we can show either errorMessage(if it is passed to params) or default message "The export is prevented until the calling entity is saved - "You do not have permissions to export"
Right?
2) if export is not permitted  -

Reply:
That's correct @vlbe. You could make both messages parameterized for the component but suspect we'll only override the `errorMessage` explicitly.